app/workers/publishing_api_worker.rb

```diff
@@ -31,11 +31,11 @@ def class_for(model_name)
   def send_item(payload, locale)
     save_draft(payload)
     Whitehall.publishing_api_v2_client.publish(payload.content_id, payload.update_type, locale: locale)
+    Whitehall.publishing_api_v2_client.patch_links(payload.content_id, links: payload.links)
   end
 
   def save_draft(payload)
     Whitehall.publishing_api_v2_client.put_content(payload.content_id, payload.content)
-    Whitehall.publishing_api_v2_client.patch_links(payload.content_id, links: payload.links)
```

Comment:
I'm surprised that we didn't have to change a test for this move. Is this worker even tested?